### PR TITLE
VACMS 14937 Add Banner to vba template

### PR DESCRIPTION
**Pull request by eselkin**
## Summary

- Adds banner dependent on CMS banner completion for a VBA Facility. Only shows the banner when form is complete on CMS (even though CMS doesn't let you save an incomplete form -- cannot test this edge case).  
- Sitewide Facilities
- No flippers

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14937

## Testing done

- Tested banner is created for informational type (unit)
- Tested banner is created for warning type (unit)
- Tested banner can be made dismissible (unit)
- e2e tests will have to be made after VBA facilities are published on prod

## Screenshots
Screenshot 2023-12-29 at 5 50 07 PM
Screenshot 2023-12-29 at 5 49 50 PM
Screenshot 2023-12-29 at 5 49 41 PM

Mobile:
Screenshot 2023-12-29 at 9 11 57 PM
Screenshot 2023-12-29 at 9 11 06 PM
Screenshot 2023-12-29 at 9 10 55 PM

## What areas of the site does it impact?

Just the VBA Regional Office pages

### Quality Assurance & Testing

- [x] I added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (not applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

View: 
- [ ] [Dismissible Info](https://web-ihqrv6rby5lxlkn8zqdxf9u7lk1aplcp.demo.cms.va.gov/cleveland-va-regional-benefit-office/locations/cleveland-va-regional-benefit-office/)
- [ ] [Dismissible Warning](https://web-ihqrv6rby5lxlkn8zqdxf9u7lk1aplcp.demo.cms.va.gov/wilmington-va-regional-benefit-office/locations/wilmington-va-regional-benefit-office/)
- [ ] [Seattle non-dismissible banner](https://web-ihqrv6rby5lxlkn8zqdxf9u7lk1aplcp.demo.cms.va.gov/vba-facilities/locations/seattle-regional-office/)

### CODE REVIEW

File: src/site/layouts/tests/vba/template/vba_banner_test.unit.spec.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @department-of-veterans-affairs/axe-check-required */
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/layouts/tests/vba/template/vba_banner_test.unit.spec.js:1]: Not able to do axe check on renderHTML test.

**Review by laflannery — APPROVED** [review]
lgtm!

**Review by randimays — APPROVED** [review]
One typo catch. Otherwise LGTM!

File: src/site/includes/vba_facilities/banner.liquid [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{% comment %} 
+              Creates a full width va-alert banner
+              expcts the following variables to be passed in
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/includes/vba_facilities/banner.liquid:3]: Thanks

[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/includes/vba_facilities/banner.liquid:3]: Actually, I have to change that comment a bit, since it's no longer a va-alert

[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/includes/vba_facilities/banner.liquid]: I just bridged the comment to make it understandable that this is a banner.

**Comment by jilladams** [comment]
Approved, tests passing, merging!

**Comment by jilladams** [comment]
Nevermind -AC for design review, gonna let @thejordanwood take a look before merging.
